scripts/functional-tests.sh

```diff
@@ -23,6 +23,7 @@ export CACHE_DIR
 export ILAB_CONFIGDIR_LOCATION
 export ILAB_CONFIG_FILE
 export ILAB_DATA_DIR
+export ILAB_DOWNLOAD_DIR
```

Comment:
Why do we need this `export` here? I think the variable is only accessed from this shell instance. (I also have doubts about the rest of variables here but that's for a separate discussion.)

Reply:
I was doing it for consistency, but you're right - I'll remove the export.

Reply:
You shouldn't need to remove the `export` as `export VAR` within a script will still keep it localized. Even if you run this script, they still won't become a part of the invoked shell's vars.

Reply:
Sure. But `export` is used when you'd like a shell / command you are calling from *this* script to inherit the variable. Since nothing outside the script actually reads the variable, the `export` is redundant.

Reply:
I see, that makes sense. I appreciate your technical insight here, it sounds like we should tackle the removal of the other `export` variables so this can be avoided in the future.

Reply:
@booxter I've created an issue to track this, thank you for bringing it to our attention: https://github.com/instructlab/instructlab/issues/1798

Reply:
@RobotSail +1 for creating a ticket. It's a good idea to address the problem, but it's not critical for 0.18.0 release. There are much bigger fish to fry.